tools/pot/openvino/tools/pot/api/samples/classification/classification_simple_api_sample.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we have some intro description at the beginning of each sample explaining its purpose, which algo is used, etc?

Reply:
There is a readme for each sample. We will update them.